Makefile

```diff
@@ -0,0 +1,54 @@
+# Set the GCP project ID: 
+# export GCP_PROJECT=<project-id>
+GCP_REGION = europe-west1
+GCP_BUCKET = dataflow-training-persistent
+GCP_TEMP_BUCKET = dataflow-training-temp
+
+directLocal:
```

Comment:
I would add a prefix `run-` to the targets which run a pipeline

Reply:
Good idea